libs/accounts/src/lib/accounts-manager.tsx

```diff
@@ -27,7 +27,7 @@ export const AccountManager = ({
   const update = useCallback(
     ({ data }: { data: AccountFields[] | null }) => {
       dataRef.current = data;
-      gridRef.current?.api.refreshInfiniteCache();
+      gridRef.current?.api?.refreshInfiniteCache();
```

Comment:
Do we really need this? Look like api is always present.
![image](https://user-images.githubusercontent.com/1754247/195796226-d63542e2-556f-4cd7-b657-d39af0cd3c1c.png)

Reply:
If it is happening that AgGrid react instance is having undefined api we should fix it also in other places.

Reply:
I just meet a run time crush there. It isn't connected with my changes, but I just wanted to fix it here. Looks like some race condition in ag-grid initialisation.  Sure, I will remove it then